app/controllers/stripe_controller.rb

```diff
@@ -0,0 +1,47 @@
+class StripeController < ApplicationController
+  protect_from_forgery except: :webhook
+
+  def webhook
+    webhook_secret = ENV["STRIPE_WEBHOOK_SIGNING_KEY"]
```

Comment:
Should this be a constant?

Reply:
Yes, it should be a constant because the value will not be changed the whole time. 
``` ruby
# stripe_controller file
def webhook
 WEBHOOK_SECRET = ENV["STRIPE_WEBHOOK_SIGNING_KEY"]
end
```

Is it better to define "WEBHOOK_SECRET" in a constants file like this? What do you think about it?
```ruby
module Constants
 WEBHOOK_SECRET = ENV["STRIPE_WEBHOOK_SIGNING_KEY"]
end

# stripe_controller file
...
Constants::WEBHOOK_SECRET
...
```

Reply:
I think define constant like that is fine. You may want to use `ENV.fetch("STRIPE_WEBHOOK_SIGNING_KEY")` if you expect it to always be set.